index.html

```diff
@@ -1702,14 +1702,14 @@ <h3>canonicalizationAlgorithm</h3>
         </pre>
       </section>
 
-      <section id="owner" about="https://w3id.org/security#owner"
+      <section id="controller" about="https://w3id.org/security#controller"
         typeof="rdf:Property">
-        <h3>owner</h3>
+        <h3>controller</h3>
         <p>
-An owner is an entity that claims control over a particular resource. Note
-that ownership is best validated as a two-way relationship where the owner
-claims ownership over a particular resource, and the resource clearly
-identifies its owner.
+A controller is an entity that claims control over a particular resource. Note
+that control is best validated as a two-way relationship where the controller
+claims control over a particular resource, and the resource clearly
+identifies its controller.
```

Comment:
We can't remove properties from this vocabulary because people may have deployed them. All we can do is deprecate properties. Please keep the section, `id`, and entry for for `owner` around... just note that it's deprecated and replaced by `controller`.